.eslintignore

```diff
@@ -1 +1,2 @@
 coverage
```

Comment:
Either one of these lines should be enough right?

Reply:
Yes. I used mrm to automate that. Not that important I'd say